R/matrix_form.R

```diff
@@ -317,13 +355,44 @@ MatrixPrintForm <- function(strings = NULL,
 
   ##  ret <- shove_refdf_into_rowinfo(ret)
   if (is.null(colwidths)) {
-    colwidths <- propose_column_widths(ret)
+    colwidths <- propose_column_widths(ret, fontspec = fontspec)
   }
   mf_col_widths(ret) <- colwidths
   ret <- mform_build_refdf(ret)
+  ret <- mpf_infer_cinfo(ret, colpaths = colpaths, fontspec = fontspec)
```

Comment:
why mpf instead of mf? (I guess we can change it if it is internal)

Reply:
Ok, I see; maybe we should make this distinction more explicit and add it to the dev guide (in another issue)